app/models/katello/glue/provider.rb

```diff
@@ -3,29 +3,11 @@ module Glue::Provider
     def self.included(base)
       base.send :include, InstanceMethods
       base.class_eval do
-        before_destroy :destroy_products_orchestration
+        before_destroy :del_products
       end
```

Comment:
So it looks like we restrict deletion of products on the relationship between provider and products. I think the only way to delete products is via the dynflow action. I am going to remove del_products and this before_destroy hook.